cmd/fence-provision-manager/cmd/fence.go

```diff
@@ -58,6 +59,13 @@ func fence(cmd *cobra.Command, args []string) (err error) {
 	}
 	fenceArgs = append(fenceArgs, fmt.Sprintf("--password=%s", password))
 
+	// Set power management action
+	action, err := cmd.Flags().GetString("action")
+	if err != nil {
+		return err
+	}
+	fenceArgs = append(fenceArgs, fmt.Sprintf("--action=%s", action))
```

Comment:
This applies for RHEL fence-agents, but how would you handle ansible playbooks for example?